okio/src/jvmMain/kotlin/okio/JvmSystemFilesystem.kt

```diff
@@ -62,7 +62,7 @@ internal open class JvmSystemFilesystem : Filesystem() {
       if (!file.exists()) throw FileNotFoundException("no such file $dir")
       throw IOException("failed to list $dir")
     }
-    return entries.map { dir / it }
+    return entries.map { dir / it }.sorted()
```

Comment:
Should we sort in place instead, since `map {}.sorted()` now copies `entries` twice?

Reply:
sure!

Reply:
Fixed.

Reply:
Same thing in FFS, but perhaps less of an issue